client/coral-framework/index.js

```diff
@@ -1,15 +1,14 @@
 import store from './services/store';
 import pym from './services/PymConnection';
 import I18n from './modules/i18n/i18n';
-import * as authActions from './actions/auth';
-import * as assetActions from './actions/asset';
-import * as notificationActions from './actions/notification';
+import actions from './actions';
+import Slot from './components/Slot';
 
-export {
+export default {
   pym,
+  Slot,
   I18n,
   store,
-  authActions,
-  assetActions,
-  notificationActions
+  actions,
+  ...actions
```

Comment:
This is a hack that we are going to address in another PR. Right now this is needed. (It's about our legacy code)